src/PlatformInstaller/Installations/ServiceControlInstaller.cs

```diff
@@ -135,7 +134,7 @@ public IEnumerable<AfterInstallAction> GetAfterInstallActions()
                 var value = GetManagementPath();
                 if (File.Exists(value))
                 {
-                    processRunner.RunProcess(value, "", Path.GetDirectoryName(value), s => { }, s => { });
+                    _ = processRunner.RunProcess(value, "", Path.GetDirectoryName(value), s => { }, s => { });
```

Comment:
Dropped task, but I don't think it matters, since this whole thing is deprecated anyway.